### DISCUSSION TITLE
Copilot shows an error when I start NoeVim

### DISCUSSION THREAD

**Discussion by bkielbasa** [community/community#40372] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I installed the copilot plugin for vim. Every time I run it I get the following error

```
               Error detected while processing /Users/bklimcza/.local/share/nvim/site/pack/packer/start/copilot.vim/plugin/copilot.vim[67]..function <SNR>46_MapTab:
line    7:
E716: Key not present in Dictionary: "rhs !~# 'copilot'"
Press ENTER or type command to continue
```

I tried to google that but no lack.
My neovim version

```
NVIM v0.9.0-dev-1621+gd311c5481-dirty
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by bklimcza@DCIL-WY4Y3WW14L-M
```

Does anyone have an idea what's wrong?

**Comment by ossanna16** [comment]
Hi there @bkielbasa and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by iamcjwilliams** [comment]
@bkielbasa Sorry for the trouble here, Our copilot team is reviewing this issue

**Comment by segeljakt** [comment]
What do you get when you run `:Copilot status`? (If possible)

**Reply by bkielbasa** [reply]
@segeljakt it says it's enabled and online

**Comment by tpope** [comment, marked as answer]
Can you tell me the output of `:echo maparg('<Tab>', 'i', 0, 1)`?

**Reply by bkielbasa** [reply]
of course!
<img width="1526" alt="image" src="https://user-images.githubusercontent.com/1235620/205867850-d1fa8783-eb36-49fb-8bdc-21b8fa301c07.png">

**Reply by tpope** [reply]
I'm not sure what's going on—that screenshot doesn't really jive with the error message—but whatever it is looks pretty easy to guard against. Try the just released 1.7.0.

**Reply by bkielbasa** [reply]
After updating to 1.7 it started working OK. Before the update, neovim started with the error I sent in my first message but after that, I got copilot's suggested code.

**Comment by tfglasswing** [comment]
I had this issue with https://github.com/zbirenbaum/copilot.lua, switched back to https://github.com/github/copilot.vim and the error went away.

```
NVIM v0.11.0
Build type: RelWithDebInfo
LuaJIT 2.1.1741730670
```

![Screenshot from 2025-04-06 15-00-58](https://github.com/user-attachments/assets/787484ec-9275-4d92-b8ba-dc000342b02d)
![Screenshot from 2025-04-06 15-01-04](https://github.com/user-attachments/assets/40c85fe7-aaf7-4c7c-b909-5cee1a10423c)

**Reply by kurund** [reply]
I am also getting same error, same plugin.

Update: It was typo at my end, see https://github.com/zbirenbaum/copilot.lua/issues/442
